core/Objects/CmfPackage.cs

```diff
@@ -547,9 +547,9 @@ public void LoadDependencies(IEnumerable<Uri> repoUris, StatusContext ctx, bool
                 }
 
                 IDirectoryInfo[] repoDirectories = repoUris?.Select(r => r.GetDirectory()).Where(d=> d.Exists==true).ToArray();
-                if (ExecutionContext.Instance.RunningOnWindows && repoDirectories.Length == 0)
+                if (ExecutionContext.Instance.RunningOnWindows && repoDirectories != null && repoDirectories.Length == 0)
                 {
-                    throw new CliException($"None of the provided repositories exist: {string.Join(", ", repoUris.Select(d => d.OriginalString))}");
+                    Log.Warning($"None of the provided repositories exist: {string.Join(", ", repoUris.Select(d => d.OriginalString))}");

```

Comment:
it's rather concerning that this change triggers no test fails...

Reply:
This exception was only being triggered on windows. When João committed this changes, he did so on a Linux system, so no tests failed here. When I tried to commit my (unrelated) changes on a Windows machine, some tests were failing because of this. Changing this makes the behavior of Windows test runs consistent with the Linux executions (which were not failing before).

Also, there are other checks down the line that fail when specific Packages are not found in any repository (which is the only reason why making this only a Warning seemed ok to us) 😄